augur/tasks/github/detect_move/core.py

```diff
@@ -44,6 +60,23 @@ def ping_github_for_repo_move(augur_db, key_auth, repo, logger,collection_hook='
         logger.warning(f"Could not check if repo moved because the api timed out 10 times. Url: {url}")
         return
 
+    
+    
+    #Mark as errored if not found
+    if response_from_gh.status_code == 404:
+        logger.error(f"Repo {repo.repo_git} responded 404 when pinged!")
+
```

Comment:
When we get 404's we should probably do at least one retry before updating the description.

Reply:
I agree I will implement that